static/css/style.css

```diff
@@ -1,546 +1,590 @@
-	justify-content: center;
-	align-items: center;
-	margin: 0rem auto;
-	max-width: 600px;
-	text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0rem auto;
+  max-width: 600px;
+  text-align: center;
 }
 
 .countdown-container .countdown-title {
-	font-weight: 600;
-	margin-bottom: 0.75rem;
-	font-size: 1.1rem;
+  font-weight: 600;
+  margin-bottom: 0.75rem;
+  font-size: 1.1rem;
 }
 
 .countdown-container .timer {
-	display: flex;
-	flex-wrap: wrap;
-	justify-content: center;
-	gap: 0.5rem;
-	font-variant-numeric: tabular-nums;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row-reverse;
+  justify-content: center;
+  gap: 0.5rem;
+  font-variant-numeric: tabular-nums;
 }
 
 .countdown-container .segment {
-	border: 1px solid #ddd;
-	padding: 0.6rem 0.75rem;
-	border-radius: 0.75rem;
-	min-width: 60px;
-	text-align: center;
-	flex: 1 1 40px;
+  border: 1px solid #ddd;
+  padding: 0.6rem 0.75rem;
+  border-radius: 0.75rem;
+  min-width: 60px;
+  text-align: center;
+  flex: 1 1 40px;
 }
 
 .countdown-container .value {
-	font-size: 1.4rem;
-	font-weight: 700;
-	display: block;
+  font-size: 1.4rem;
+  font-weight: 700;
+  display: block;
 }
 
 .countdown-container .label {
-	font-size: 0.75rem;
-	color: #555;
+  font-size: 0.75rem;
+  color: #555;
 }
 
 .countdown-container .live-badge {
-	display: none;
-	justify-content: center;
-	gap: 0.5rem;
-	font-weight: 700;
-	color: #2e9c6a;
-	margin-top: 0.5rem;
-	font-size: 1rem;
+  display: none;
+  justify-content: center;
+  gap: 0.5rem;
+  font-weight: 700;
+  color: #2e9c6a;
+  margin-top: 0.5rem;
+  font-size: 1rem;
 }
 
 .countdown-container .live-dot {
-	width: 0.6rem;
-	height: 0.6rem;
-	border-radius: 999px;
-	background: #2e9c6a;
+  width: 0.6rem;
+  height: 0.6rem;
+  border-radius: 999px;
+  background: #2e9c6a;
 }
 
 table {
-	width: 100%;
-	border-collapse: collapse;
+  width: 100%;
+  border-collapse: collapse;
 }
 
 th,
 td {
-	padding: 12px 16px;
-	text-align: right;
-	border-bottom: 1px solid #eee;
+  padding: 12px 16px;
+  text-align: right;
+  border-bottom: 1px solid #eee;
 }
 
 thead th {
-	background-color: #f5f5f5;
-	font-weight: bold;
+  background-color: #f5f5f5;
+  font-weight: bold;
 }
 
 tr:hover td {
-	background-color: #f9f9f9;
+  background-color: #f9f9f9;
 }
 
 th:first-child,
 td:first-child {
-	text-align: center;
-	width: 50px;
+  text-align: center;
+  width: 50px;
 }
 
 .empty-row td {
-	text-align: center;
-	color: #888;
-	font-style: italic;
-	background: #fafafa;
+  text-align: center;
+  color: #888;
+  font-style: italic;
+  background: #fafafa;
 }
 
 footer {
-	background: #f3f4f6;
-	text-align: center;
-	padding: 1rem;
-	font-size: 0.85rem;
-	border-top: 1px solid #e5e7eb;
-	line-height: 1.6;
-	color: #4b5563;
+  background: #f3f4f6;
+  text-align: center;
+  padding: 1rem;
+  font-size: 0.85rem;
+  border-top: 1px solid #e5e7eb;
+  line-height: 1.6;
+  color: #4b5563;
 }
```

Comment:
all in all, i like the color pallete -- esp the background color and the fonts. but i think the links and lines look so light and they hurt my eyes a little :) let's change them to sth more dim